packages/apps-config/src/api/spec/index.ts

```diff
@@ -97,7 +97,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   Pangolin: pangolin,
   VLN: vln,
   'VLN-PC': vlnrococo,
-  acala,
+  ...acala,
```

Comment:
The issue is that above you export an `OverrideBundleDefinition` and here you expand it. This is why the build fails.

Reply:
fixed the type assertion